ingest/src/commands/pages.ts

```diff
@@ -1,15 +1,43 @@
 import { CommandModule } from "yargs";
 import { updatePages } from "../updatePages";
+import { makeSnootyDataSource } from "../SnootyDataSource";
+import { makeDevCenterDataSource } from "../DevCenterDataSource";
+import { snootyProjects } from "../snootyProjects";
 
 // TODO: Option for which data source
 type PagesCommandArgs = Record<string, never>;
```

Comment:
to clarify my understanding: are you not yet implementing passing the data source as args? and right now it's hard coding the data sources in the command itself? 

fine for now, but could you create a ticket for the data sources as args, and note it in a comment?

Reply:
https://jira.mongodb.org/browse/DOCSP-30988